src/CONST.js

```diff
@@ -1641,6 +1664,61 @@ const CONST = {
         ZW: 'Zimbabwe',
     },
 
+    ALL_US_STATES: {
```

Comment:
I see we are just using the keys `_.keys` maybe we can change it to be just a list of ALL_US_ISO_STATES